wgengine/userspace.go

```diff
@@ -1554,3 +1557,8 @@ func (ls fwdDNSLinkSelector) PickLink(ip netaddr.IP) (linkName string) {
 	}
 	return ""
 }
+
+var (
+	metricMagicDNSPacketIn = clientmetric.NewGauge("magicdns_packet_in") // for 100.100.100.100
+	metricReflectToOS      = clientmetric.NewGauge("packet_reflect_to_os")
```

Comment:
Suggest `wgengine_packet_reflect_to_os`. The `tstun` namespacing above seems like a thing worth doing more of.

Reply:
I didn't do it here because it's not really a function of wgengine at a design level. It's a weird hook installed from afar and really it could be done as part of tstun. The exact place it's installed today isn't incredibly relevant.

In the prefixed cases earlier it's a core function of that package.